Core/scripts/dirac-install.py

```diff
@@ -786,7 +786,7 @@ def urlretrieveTimeout( url, fileName = '', timeout = 0 ):
     # Sometimes repositories do not return Content-Length parameter
     try:
       expectedBytes = long( remoteFD.info()[ 'Content-Length' ] )
-    except Exception, x:
+    except Exception as x:
```

Comment:
This syntax is only supported in python 2.6+, so it would be good to not introduce this in dirac-install and in scripts called during pilot initialization.